### PR TITLE
Add PHP Page

### PR DESCRIPTION
**Pull request by robinrodricks**
Adds the PHP getting started guide under platforms

### CODE REVIEW

**Comment by Simn** [comment]
Please create distinct branches for your pull requests. Otherwise this ends up being really messy.

In general, I have to wonder if we want to go so deep into the target-specifics. Is it really Haxe's responsibility to document how to set up your PHP run-time? I mean, it's certainly helpful information, but at the same time it is also more content we have to maintain and keep up-to-date.

Don't take this the wrong way please. I appreciate your work and it's good content. I'm just having concerns about this being a bit out-of-scope for our website.

**Comment by robinrodricks** [comment]
Well, my basic idea was helping to onboard developers. Currently haxe is pointless without platform specific info, and even if it gets out of date, having one version of a guide is better than having none.

File: pages/documentation/platforms/php.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+		Develop your project
+	</h2>
+	<div class="step-info">
+		<h4>Haxe project</h4>
+		<p>
+			Refer to <a href="/manual/target-php-getting-started.html">this guide</a> to create a basic Haxe PHP project.
+		</p>
+		<h4>Haxe libraries</h4>
+		<p>
+			You can use <a href="https://haxetink.github.io/#/">Tinkerbell</a> to boost your development. <br/>
```
Comment (markknol) [inline pages/documentation/platforms/php.html]: I don't think it's needed to promote certain libraries here

**Comment by markknol** [comment]
@RealyUniqueName can you also take a look if the content is correct?

**Comment by markknol** [comment]
@robinrodricks Thanks, I'm happy that you want to do this. If content is correct, I can merge this, but you really need to re-open the PR from another branch (not staging).

**Comment by robinrodricks** [comment]
Okay Mark, I'll create another branch and remove the reference to
Tinkerbell. It's a bit sad though coz no one is gonna know about it and
based on the reviews it got in the Haxe Survey it seems many people find it
indispensable.

**Comment by RealyUniqueName** [comment]
Great contribution! Thanks!
Here is what I think needs to be changed:

> Install a PHP runtime

This is not a requirement. One can deploy generated PHP code to a server instead of running it locally.  
"Run your project" section should be adjusted accordingly.

>  Refer to <a href="/manual/target-php-getting-started.html">this guide</a> to create a basic Haxe PHP project.

I think that page should become a part of the new page. Otherwise, we will have two "getting started" pages for PHP.

> Browse the <a href="https://api.haxe.org/">Haxe API website</a> for the core Haxe platform API that you can use in your project, which replaces all PHP methods that you would typically use in a PHP project.

This is not correct: "which replaces all PHP methods that you would typically use in a PHP project".  
We don't have a complete set of PHP externs in std library yet. 
It's better to remove this part of the sentence.

Also, I think it's worth mentioning that Haxe 3.4 generates code for PHP 5.4+ by default and for 7.0+ with `-D php7` flag. 
While Haxe 4 generates code for PHP 7.0+ by default.

**Comment by robinrodricks** [comment]
Are we good now?

**Comment by Simn** [comment]
You will still have to reopen this PR from a different branch. There are unrelated changes to charp.html which should not be here. Please send the PR from robinrodricks:phppage or something like that.

**Pull request closed without merging**